### ISSUE TITLE
Add ability to add ENV variables

### ISSUE THREAD

**Issue by agentmilindu** [leopardslab/Dunner#2]
The users should be able to set ENV variables to the Docker containers from `.dunner.yaml`

```
- name: 'mesosphere/aws-cli'
  command: ['aws', 'elasticbeanstalk update-application --application-name myapp']
  envs: 
   - AWS_ACCESS_KEY_ID=8O7GO86F87GY
   - AWS_SECRET_ACCESS_KEY='767tg87to867TRI68ri76R87To867T8I6rbO87T&$@bo87Tbo87' 
   - AWS_DEFAULT_REGION=us-east1
```

**Comment by vaibhavk** [comment]
I would like to work on this issue

**Comment by ayushjn20** [comment]
I would like to work on it.

**Comment by agentmilindu** [comment]
FIxed with #43
